pandas/io/parquet.py

```diff
@@ -167,6 +167,10 @@ def write(
     ):
         self.validate_dataframe(df)
 
+        # PyArrow does not support saving float16
```

Comment:
comment is superfluous

Reply:
can you add a more meaningful message e.g. `columns [.....]` have this dtype.

Reply:
Thanks, removed the comment and changed the message to include column names with dtype as float 16.